demo.py

```diff
@@ -28,12 +30,17 @@ def download_data():
 
 # demonstration - this will be removed later
 if __name__ == "__main__":
+
     print(sys.version)
     print(sys.executable)
 
     # Download example data (if it doesn't exist)
     download_data()
 
+    dataframe_dict = {'airlines': pd.read_csv('data/airlines/airlines.csv'),
```

Comment:
Keep in mind this mock-up will only work on a Mac or linux.  If someone using a PC runs this code it won't work because the path names need to be reversed.